index.html

```diff
@@ -18,19 +18,19 @@
 </div>
 <div class="wff_90">
 <span class="t_hm">our products</span>
-<span class="t_h1">Fix server security with step-by-step instructions</span>
-<span class="t_h2">Get security alerts delivered to your Inbox or Slack as they are discovered. Turn your DevOps team into a DevSecOps team.</span>
+<span class="t_h1">A lightweight agent installed on your server protects your data and workloads</span>
```

Comment:
I think this needs to be shrunk a bit.

Reply:
Updated the branch. I've shortened it from the first proposed change but it's still a bit longer than the previous line. I think it'll be ok looking at the current site layout.